### DISCUSSION TITLE
Just upgraded to MobiFlight 9.7.0;  MOBIFLIGHT_XXX events don't work anymore

### DISCUSSION THREAD

**Discussion by Rascal650** [nguyenquyhy/Flight-Tracker-StreamDeck#387] [General]
Flight tracker was working fine for me until today.  I"m using  MOBIFLIGHT events with the FBW320.  

I upgraded to Mobiflight 9.7.0 and none of my MOBIFLIGHT_XXX buttons are working anymore.  Using a different profile and plane (Cesna 172), I tried non-mobfilight  (like TOGGLE_MASTER_ALTERNATOR or TOGGLE_MASTER_BATTERY) commands and they still work fine.  So the problem seems to affect MOBIFLIGHT events only. 

I reinstalled Flight Tracker plug-in but the MOBIFLIGHT commands are still broken.   

Is there  a log fine that I can attach?

**Comment by Rascal650** [comment]
UPDATE:  I got it fixed.  

After upgrading to MobiFlight 9.7.0,  I did the following (not sure if all the steps are needed)
1-  Extras -> HubHop -> Download Latest Presets
2 - Extras ->  MSFS2020 -> Update events.txt (legacy)
3- Restart MSFS

The MOBIFLIGHT_XXX events are running again.

I'm a bit concerned about the "legacy" status of events. txt in the menu.  Does that mean that Mobiflight will soon stop supporting events.txt and that FlightTracker won't be able to use MOBIFLIGHT commands?

**Reply by nguyenquyhy** [reply]
I have just added a more detailed guide on MobiFlight integration. Hopefully, it can help new comers with the setup.
https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/blob/main/docs/MOBIFLIGHT.md

Regarding the legacy status, I don't really know what the future holds. We can always talk to MobiFlight to find a solution if they end up removing the feature.
